groups/voteForDelegate.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
+const logger = require('../helpers/logger');
+const keys = require('../helpers/keys');
+const constants = require('../helpers/constants');
+const transactionFormer = require('../helpers/transactionFormer');
+const validator = require('../helpers/validator');
+
+const DEFAULT_SEND_TOKENS_RETRIES = 4; // How much re-tries for send tokens requests by default. Total 4+1 tries
+
+module.exports = (nodeManager) => {
+  /**
+		* Creates votes for delegate transaction, signs it, and broadcasts to ADAMANT network
+    * See https://github.com/Adamant-im/adamant/wiki/Transaction-Types#type-3-vote-for-delegate-transaction
+    * @param {string} passPhrase Senders's passPhrase. Sender's address will be derived from it.
+    * @param {string[]} votes PublicKeys for upvote and downvote
```

Comment:
Allow votes[] to include publicKeys and ADM addresses and delegate names (mixed).
Transform ADM addresses and delegate names to publicKeys.
Exclude duplicates.